include/COSMO_SAC/profile_db.hpp

```diff
@@ -346,6 +346,83 @@ class DelawareProfileDatabase : public ProfileDatabase {
     }
 };
 
+/// The DirectImport feature for importing profiles based on "filename" + "path" 
+class DirectImport : public ProfileDatabase {
+private:
+    std::string m_dbpath;
+
+public:
+    DirectImport() {
+    }
+
+    void add_profile(const std::string& identifier, const std::string& dbpath = ".") {
+        m_dbpath = dbpath;
+        // Now we load the sigma profile(s) from the file
+        auto lines = str_split(get_file_contents(m_dbpath + "/" + identifier + ".sigma"));
+
+        std::vector<double> _sigma, _psigmaA;
+        FluidProfiles fluid;
+        std::string meta;
+        for (auto &&line : lines) {
+            if (line.substr(0,8) == "# Name: "){ std::string check_name = line.substr(8,line.size()-8); continue;}
```

Comment:
The second argument to substr is not needed if you are keeping everything to the end of the string.